rootfs/opt/kubectl/example/GoogleBackup/manifest/statefulset-backup.tpl.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: <TIER>
+  labels:
+    tier: <TIER>
+    role: backup
+    app: qserv
+spec:
+  selector:
+    matchLabels:
+      tier: qserv
+      role: backup
+      app: qserv
+  serviceName: qserv
```

Comment:
I think this ends up making the name of the pods - maybe call this qserv-backup or something, just to keep it straight from qserv?

Reply:
The label "app: qerv" is required for backup pods to attach Qserv PVC. The serviceName is used for DNS, not for naming pods.